### ISSUE TITLE
Ctrl+W doesn't remove tab from sidebar

### ISSUE THREAD

**Issue by DarylPinto** [betterbrowser/arcfox#79]
While pressing Ctrl+W closes a tab properly (as it is no longer accessible), a "ghost" tab stays present in the sidebar. This ghost tab cannot be selected.
___

Browser
```
Firefox 129.0.1 (64-bit)
```

OS
```
Edition           Windows 11 Pro
Version           23H2
OS build          22631.4037
Experience        Windows Feature Experience Pack 1000.22700.1027.0
```

**Comment by LuanderFarias** [comment]
Thanks for the review. We gonna fix that

**Comment by GuiMar10** [comment]
Getting fixed on ArcFox v2.5.3 ;)
